src/core/Select/Select.tsx

```diff
@@ -237,14 +245,14 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
           key: `${option.label}-${index}`,
           isSelected,
           disabled: option.disabled,
-          value: option.value,
           onClick: () => {
             !option.disabled && onChange?.(option.value);
             close();
           },
           ref: (el: HTMLElement) => isSelected && el?.scrollIntoView(),
           icon: option.icon,
           role: 'option',
+          ...option,
```

Comment:
didnt want to add props? so then maybe remove all above.. idk..

Reply:
Removed all